### PR TITLE
[BASE]: improve access rights changes.

### PR DESCRIPTION
**Pull request by atp-odoo**
[REM]Marketing: Remove Marketing groups- user/manager
[ADD]Marketing: add 'lead automation' and 'Mass mailing' groups;
[ADD]UTM: Put "Administration/configuration" right in "Link Tracker -> UTMs" 
MOved to Technical group : Contact Creation (keep true)
Portal,Public,Multi Company groups.

Task: https://www.odoo.com/web?#id=19788&view_type=form&model=project.task&action=327
Pad: https://pad.odoo.com/p/r.P05YP1cyr3wEjbR6

### CODE REVIEW

File: addons/fleet/security/fleet_security.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <data noupdate="0">
         <record model="ir.module.category" id="module_fleet_category">
             <field name="name">Fleet</field>
-            <field name="sequence">17</field>
+            <field name="sequence">16</field>
```
Comment (atp-odoo) [inline addons/fleet/security/fleet_security.xml]: try to use minimal change on sequence.

File: addons/marketing/security/marketing_security.xml [hunk 2]
```diff
@@ -1,15 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
 <data>
-    <record id="marketing.group_marketing_user" model="res.groups">
+
+    <record model="ir.module.category" id="module_category_lead_automation">
+        <field name="name">Lead Automation</field>
+        <field name="sequence">3</field>
+        <field name="description">Helps you manage lead of your marketing campaigns step by step.</field>
+    </record>
```
Comment (atp-odoo) [inline addons/marketing/security/marketing_security.xml]: create this category in base/module/module_data.xml file.

File: addons/mass_mailing/security/mass_mailing_security.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="module_category_mass_mailing" model="ir.module.category">
+        <field name="name">Mass Mailing</field>
+        <field name="sequence">11</field>
+        <field name="description">Helps you manage your mass mailing for design
+            professional emails and reuse templates.</field>
+    </record>
```
Comment (atp-odoo) [inline addons/mass_mailing/security/mass_mailing_security.xml]: create category into base/module/module_data.xml

File: addons/subscription/security/ir.model.access.csv [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
-access_subscription_subscription_user,subscription.subscription user,model_subscription_subscription,base.group_tool_user,1,1,1,1
-access_subscription_subscription_history_user,subscription.subscription.history user,model_subscription_subscription_history,base.group_tool_user,1,1,1,1
-access_subscription_document_user,subscription.document user,model_subscription_document,base.group_tool_user,1,1,1,1
-access_res_partner_user,res.partner.user,base.model_res_partner,base.group_tool_user,1,1,1,1
-access_ir_cron_user,ir.cron.user,base.model_ir_cron,base.group_tool_user,1,1,1,1
+access_subscription_document_fields,subscription.document.fields,model_subscription_document_fields,,1,1,1,1
+access_subscription_subscription_user,subscription.subscription user,model_subscription_subscription,,1,1,1,1
+access_subscription_subscription_history_user,subscription.subscription.history user,model_subscription_subscription_history,,1,1,1,1
+access_subscription_document_user,subscription.document user,model_subscription_document,,1,1,1,1
+access_res_partner_user,res.partner.user,base.model_res_partner,,1,1,1,1
```
Comment (atp-odoo) [inline addons/subscription/security/ir.model.access.csv]: it should be read access. 
like this
access_res_partner_user,res.partner.user,base.model_res_partner,,1,0,0,0

**Comment by gan-odoo** [comment]
@atp-odoo  I made all changes as per your comments.
Thanks.

**Comment by tde-banana-odoo** [comment]
Just pushed a cleaned version in https://github.com/odoo/odoo/pull/8013 .

**Pull request closed without merging**